### DISCUSSION TITLE
pop over on disabled button

### DISCUSSION THREAD

**Discussion by lewysigns** [anvilistas/anvil-extras#338] [Q&A] [answered]
Is there a way to make the pop over appear on a disabled button?

This can be seen using the example given on the anvil forum : [https://popover-example.anvil.app](url)

Disable the top button, scroll to remove the pop over, then try to hover to view the pop over.

Use case:

I have buttons disabled and want to use pop-overs to let the user know why it is disabled.

Thanks!

**Comment by s-cork** [comment, marked as answer]
Could you try putting the button inside a flow panel and applying the popover to the flow panel instead?

If the padding on the flow panel becomes annoying change the spacing above and below to none. And then you may need to add a role on the flow panel to set the margin and padding to 0.

**Reply by lewysigns** [reply]
That does work and you do need to change the margin a padding to 0!

Thanks Stu!
